responses/sk/HassTurnOn.yaml

```diff
@@ -2,8 +2,10 @@ language: sk
 responses:
   intents:
     HassTurnOn:
-      default: Turned on {{ slots.name }}
-      lights_area: Turned on lights in {{ slots.area }}
+      # musi sa vyriesit sklonovanie, navrhujem pouzivat "Zapol som | Zapinam ....."
+      default: "Zapínam {{ slots.name }}"
+      lights_area: "Svetlá v {{ slots.area }} boli rozsvietené"
```

Comment:
svetlá v ... bola rozsvietená? - není to takto gramaticky správe? - fakt nevím - vycházím z CZ

Reply:
Spravne je ... boli rozsvietene :) Ale dik za postreh.

Reply:
Ok, tak to máte jinak :D